eslint/babel-eslint-plugin-development/src/index.js

```diff
@@ -2,7 +2,7 @@ import noDeprecatedClone from "./rules/no-deprecated-clone";
 import noUndefinedIdentifier from "./rules/no-undefined-identifier";
 import pluginName from "./rules/plugin-name";
 
-export default {
+module.exports = {
```

Comment:
🤔Shouldn't eslint refers to the built version `lib/index.js`?

Reply:
Yeah, it does! Since ESLint core doesn't have knowledge of the interop it doesn't look for `exports.default`. Leaving `module.exports` here means it won't transform it in `lib/index.js`.